src/services/dashboards/DashboardsContainer.vue

```diff
@@ -30,6 +32,12 @@ export default {
     components: { DashboardsLNB, GeneralPageLayout, VerticalPageLayout },
     setup() {
         const { breadcrumbs } = useBreadcrumbs();
+
+        /* init */
+        (async () => {
+            await store.dispatch('display/loadCurrencyRates');
+        })();
```

Comment:
suggestion: How about get currencyRates in `DashboardWidgetContainer.vue`?

Reply:
Ah.. that seems more efficient.